guides/common/modules/proc_upgrading-a-package.adoc

```diff
@@ -4,9 +4,10 @@
 Use the following procedure to view the installed packages of a host.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and select the name of the host that contains the package you want to upgrade.
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the name of the host that contains the package you want to upgrade.
 . On the *Content* tab, select *Packages*.
-. The *Status* column displays whether the package is upgradable or Up to date.
++
+The *Status* column displays whether the package is upgradable or Up to date.
```

Comment:
Thanks for the screenshot, should this be "Up-to date", then? Just in case someone tries a full-text search.

Reply:
Yes, applied.